### DISCUSSION TITLE
External XJT Lite module not working on BETAFPV LiteRadio 3 Pro

### DISCUSSION THREAD

**Discussion by MKUltra19** [EdgeTX/edgetx#3805] [Announcements]
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### What part of EdgeTX is the focus of this bug?

Transmitter firmware

### Current Behavior

It is possible to choose "XJT" as external module but it's not working. Even the LED of the XJT module won't light up.
Also saw someone on Youtube with the same issue. Maybe it's the Code for the XJT module not for the XJT Lite module?
Since the Radio has a nano bay you can only use the XJT Lite module.

Link to the Video:

https://www.youtube.com/watch?v=Y3JBnwtLXzY

### Expected Behavior

Working XJT module since the Radio is also sold as "supports XJT"

### Steps To Reproduce

Works everytime

### Version

2.8.0

### Transmitter

BetaFPV LiteRadio3 Pro

**Comment by raphaelcoeffic** [comment]
@MKUltra19 the LiteRadio 3 Pro does not have the necessary hardware connection to run that module. In fact, very few radios do. You best chance would be to use a MPM module instead.

**Reply by MKUltra19** [reply]
Thank you! I ordered one :)

**Comment by dexif** [comment]
I also bought it because I was sure it would work. Why didn't I find this issue before buying it? :(
